### ISSUE TITLE
Patch did not work in bottles (A wine wrapper for linux to run windows game)

### ISSUE THREAD

**Issue by yingshaoxo** [ManlyMarco/HS2-HF_Patch#23]
Bottles: https://usebottles.com

___

Honey select original version is working fine after I installed mono extension.

But it won't load your patch automatically when I start it.

___

According to my check, when I install this patch, it keep tell me the japanese path and chinese path file can not get found.

Even if I installed japanese and chinese in deepin linux, it is still can not found those files that has japanese or chinese inside. It might be a bug for 'bottles'.

The biggest effecting is all those honey_select_2 extension can't get automatically start because of the japanese or chinese path reason. (I don't know how to start them manually)

In virtualbox, everything is working fine but super slow.

I'm not the author of those extension or patchs, otherwise, I will do everything in English.

___

That is what I found so far, thank you for reading.

**Comment by ManlyMarco** [comment]
Move the game to a directory without any Japanese characters in the path.

**Comment by yingshaoxo** [comment]
> Move the game to a directory without any Japanese characters in the path.

My system is English system, does not have any japanese characters.

"/home/yingshaoxo/Documents/honey_select_2"

**Comment by yingshaoxo** [comment]
> Move the game to a directory without any Japanese characters in the path.

By the way, you should try `bottles` in Linux, otherwise, you won't understand this wired problem.

You can mount your ntfs windows disk into linux media folder by using `sudo mount /dev/sda5 /media/game`

Then allow bottles to access it by `sudo flatpak override --filesystem=/media/game com.usebottles.bottles`

___

You can install bottles by using:
```
sudo apt install flatpak
sudo flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo && sudo flatpak remote-modify flathub --url=https://dl.flathub.org/repo/

sudo flatpak install flathub org.gnome.Platform.Compat.i386//42 org.freedesktop.Platform.GL32.default//21.08 org.freedesktop.Platform.GL.default//21.08 org.freedesktop.Platform.VulkanLayer.MangoHud com.valvesoftware.Steam.Utility.gamescope


flatpak install flathub com.usebottles.bottles
flatpak run com.usebottles.bottles
```

https://flathub.org/apps/com.usebottles.bottles

https://hu60.cn/q.php/bbs.topic.103415.html

**Comment by ManlyMarco** [comment]
Honestly, this is super low priority to me. Whenever I try out using linux as a desktop system I guess.

**Comment by yingshaoxo** [comment]
Thank you for saying the trying word.

I actually have a super convenient method for you to try linux desktop system.

I recently found Linux can get boot in uefi or bios computer from USB directly, but windows can't. I am not talking about PE system, it is a system you can save files.

You can papare a 16GB or 32GB or 64GB USB, format it to ext4, then install deepin 20.9 version directly to the USB, then when you boot from computer, you manaully choose where to boot, it might be a .efi file called 'deepin'. (But for me, I use a 512GB USB SSD, because it is quicker.)

> You have to disable secure boot in bios in your computer, and so on... Hope you know what I was talking about.

> You actually need 2 USB, one is for install deepin boot system as 'usb system install usb', another is the place you want to install the real deepin system in.

> Deepin system download: 'https://www.deepin.org/en/download' or 'https://cdimage.deepin.com/releases/20.9/'

**Comment by leucome** [comment]
I use HF patch on Linux with wine,  it usually work. So the bug  is probably specific to your system or Deepin.  Maybe you could try with an other distro to see if it work. Then compared the two to find what is missing. 

There are thing that are known to affect the patch, Like loading winhttp.dll injector is required for bepinex. 
Run winecfg then add winhttp.dll in libraries section after set it to "native then builtin".  

Also, I noticed that some version of wine staging can fail to run the patch properly,   So try with base wine or proton to to make sure it not the wine staging bug.

**Comment by yingshaoxo** [comment]
@leucome 

you are right. recently i found new wine version has a lot of made bugs. on my ubuntu14, wine1.2.6 version almost can run any x86 xp i32 software. but in new wine version, a lot of bugs will happen. some of them (such as android winlator) even deleted 'wineconsole', without 'wineconsole', the git shell will fail to get user input and stuck the system. some of them (aarch64 debian) even deleted the wine32 package from apt mirror to prevent user to run xp software...
